### PR TITLE
Testing CI

### PR DESCRIPTION
**Pull request by prashanth26**
This is a sample PR to test the working of CI

### CODE REVIEW

**Comment by rfranzke** [comment]
Hey, can you please use [this](https://github.com/gardener/gardener/blob/master/.ci/test#L1-L35) at the beginning of your `test` script?

**Comment by prashanth26** [comment]
Sorry about the late reply. I had created this PR to just check the working of the PR build pipeline. But sure, I will add the mentioned script at the beginning of our test script. Thanks :)

File: .ci/integration-tests [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+    printf "\n\t%s" "$output"
+}
+
+function hf_object_delete() {
+    output=$(kubectl delete -f "$Objects_path"/"$1" --context=control 2>&1)
+    outputSucccess=$(echo "$output" | grep deleted -c)
+    if [[ outputSucccess -ne 1 ]]; then
+        printf "\tFailed: To delete object. Exiting Test to avoid further conflicts.\n"
+        printf "\tDelete error message: %s" "$output"
+        terminate_script
```
Comment (hardikdr) [inline .ci/integration-tests]: We will improve the cluster clean-up in next PR.

File: .ci/sample-objects/aws/md-scale-down.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Sample machine-deploy object
+
+apiVersion: machine.sapcloud.io/v1alpha1
```
Comment (hardikdr) [inline .ci/sample-objects/aws/md-scale-down.yaml]: While integrating other providers, let's try to move this samples as part of the pre-created test-shoot-clusters.

**Pull request merged**